### DISCUSSION TITLE
Automatic HTTPS Redirection and SSL Certificate Configuration for dokploy Instances

### DISCUSSION THREAD

**Discussion by axadrn** [Dokploy/dokploy#278] [Q&A]
Description:

I am running a dokploy instance on a Hetzner server with several domains pointing to it. I need help with automatic HTTPS redirection and SSL certificate configuration.

Situation:

	•	Domains are registered with Porkbun.
	•	Primary domain (dokploy.example.com) points to the dokploy instance.
	•	Domains like app.com and www.app.com point to different apps on the dokploy instance.
	•	Automatic HTTPS redirection works for app.com and www.app.com with certificates set to “none”.
	•	No automatic HTTPS redirection for the primary domain (dokploy.example.com) unless https:// is explicitly used.

Observation:

In dokploy, under “Route Settings” -> “Server” -> “Server Domain”, when I set my sub-domain and choose “Certificate: None”:

	•	HTTP version is accessible but not secure.
	•	HTTPS version works with Porkbun’s SSL certificate but requires https:// to be explicitly typed.

For project domains (example.com and www.example.com), with “Certificate: None” and HTTPS enabled:

	•	example.com automatically redirects to HTTPS.
	•	www.example.com automatically redirects to HTTPS.

Question:

Why does automatic HTTPS redirection work for project domains but not for the server domain? Should I handle SSL certificates and automatic HTTPS redirection on dokploy’s side (e.g., using Traefik) or on Porkbun’s side?

Additional Information:

	•	Traefik is used as a reverse proxy on the dokploy instance.
	•	Let’s Encrypt certificates are configured in dokploy but not active for all domains.

Thank you for your help!

**Comment by Siumauricio** [comment]
Hi, In the latest version we fixed the redirection from http to https, can you check if you still have the problem?

about your dokploy server domain redirection I think we should expose the certificate option and the https toggle on our side.

**Reply by axadrn** [reply]
Thanks for your feedback!

**Comment by axadrn** [comment]
closed
